cobc/cobc.h

```diff
@@ -95,14 +95,17 @@
 /* Source format enum */
 enum cb_format {
 	CB_FORMAT_FIXED = 0,
+	CB_FORMAT_COBOL85,	/* Fixed format with Area A checks (pending) */
```

Comment:
While we're here... both CB_FORMAT_FIXED and CB_FORMAT_VARIABLE should get a comment referencing COBOL 2002+ fixed-form/free-form and the comment for the enum changed to "Source reference-format enum"